boards/arm/ti_am62x_sk_m4/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _ti_am62x_sk_m4:
+
+AM62x-SK M4F Core
+#################
+
+Overview
+********
+
+The AM62x-SK board configuration is used by Zephyr applications that run on
+the TI AM62x platform. The board configuration provides support for the ARM
+Cortex-M4F CPU and the following devices:
+
+- Nested Vector Interrupt Controller (NVIC)
```

Comment:
I don't think this comes under "having support for following devices".

NVIC is used to handle all the interrupts and exceptions that Cortex-M supports, I'd rather call it as "supported features" or something on those lines?

Reply:
Changed it to features from devices.